data/json/effects_on_condition/mutation_eocs/mutation_effect_eocs.json

```
@@ -685,5 +685,16 @@
     "//": "will remove ink_gland_ink effect when the mutation is lost",
     "condition": { "and": [ { "u_has_effect": "ink_gland_ink" }, { "compare_string": [ "INK_GLANDS", { "context_val": "trait" } ] } ] },
     "effect": [ { "u_lose_effect": "ink_gland_ink" } ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_insect_wings_shutoff",
+    "recurrence": "1 seconds",
+    "condition": { "math": [ "u_val('stamina')", "<", "3000" ] },
+    "deactivate_condition": { "not": { "u_has_trait": "WINGS_INSECT_active" } },
+    "effect": [
+      { "u_activate_trait": "WINGS_INSECT_active" },
+      { "u_message": "You don't have the stamina to keep buzzing.", "type": "bad" }
+    ]
```

```suggestion
    "effect": { "u_activate_trait": "WINGS_INSECT_active" },
    "false_effect": { "u_message": "You don't have the stamina to keep buzzing.", "type": "bad" }
```

If stamina > 3000, activate trait; If less than 3000, print message

---

It's supposed to be, if stamina > 3000, do nothing. If less than 3000, activate WINGS_INSECT_active, which turns it off. It was working fine before your initial suggestion, so I'm going to go back to what I had unless there's a good reason not to.

---

it seems i completely miss what this EoC should do then

---

Oh, i see
I'll change it when it would be merged, would be easier to test stuff for me